app/assets/javascripts/common_chart_options.js

```diff
@@ -131,6 +131,9 @@ function barColumnLine(d, c, chartIndex, seriesData, chartType, noZoom) {
       } else if (y2AxisLabel == 'Degree Days') {
         axisTitle = 'Degree days';
         pointFormat = '{point.y:.2f} Degree days';
+      } else if (isAStringAndStartsWith(y2AxisLabel, 'Carbon Intensity of Electricity Grid')) {
```

Comment:
Agreed on your point before - at some point we need to refactor this stuff and make a nicer way of handling Y2Axis stuff